### PR TITLE
Added script to update chatops integration setting

### PR DESCRIPTION
**Pull request by yypptest**
Added the script chatops-update.sh to help customer update integration settings for Chatops.

### CODE REVIEW

**Comment by yypptest** [comment]
@stacyped ^^

File: chatops-update.sh [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    echo ""
+    while getopts ":t:a:s:u:n:p:h" opt
+    do
+        case $opt in
+            t)
+            new_slack_token=$OPTARG
+            echo "Slack hubot token HUBOT_SLACK_TOKEN:  $new_slack_token"   
+            prefix="xoxb"
+            if [[ $new_slack_token != $prefix* ]];then
+                echo "ERROR: The Slack Hubot token format is not correct, it should start with xoxb "  | tee -a "$logpath"
```
Comment (stacyped) [inline chatops-update.sh:38]: should or must?

File: chatops-update.sh [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                exit 1
+            fi
+            ;; 
+            a)
+            new_pd_apikey=$OPTARG
+            echo "PagerDuty REST API key:  $new_pd_apikey"  
+            string_lenth=20
+            if [[ ${#new_pd_apikey} -ne $string_lenth ]];then
+                echo "ERROR: The PagerDuty REST API key format is not correct, pls double check it"  | tee -a "$logpath"
```
Comment (stacyped) [inline chatops-update.sh:47]: The PagerDuty REST API key format is not correct. (can we tell them what to check?)

File: chatops-update.sh [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+            s)
+            new_pd_servicekey=$OPTARG
+            echo "PagerDuty service key (integration key):  $new_pd_servicekey" 
+            ;;
+            u)
+            new_cem_url=$OPTARG
+            echo "IBM Cloud Pak console URL:  $new_cem_url" 
+            prefix="https://"
+            if [[ $new_cem_url != $prefix* ]];then
+                echo "ERROR: The Monitoring service URL is not correct, it should start with https"  | tee -a "$logpath"
```
Comment (stacyped) [inline chatops-update.sh:60]: must start with?

File: chatops-update.sh [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+            *)
+            print_help
+            exit 1
+            ;;
+        esac
+    done
+
+    if [ -n "$new_cem_url" ]||[ -n "$new_cem_apikey" ]||[ -n "$new_cem_apipass" ]||[ -n "$new_pd_apikey" ]||[ -n "$new_pd_servicekey" ]||[ -n "$new_slack_token" ]; then
+        echo ""
+        echo "Do you want to continue with above inputs [ y or n; \"n\" is default ]?"
```
Comment (stacyped) [inline chatops-update.sh:85]: Do you want to continue with these inputs

File: chatops-update.sh (hunk 1 above)
Comment (yypptest) [reply chatops-update.sh:38]: The Slack Hubot token is a string start with xoxb, so this error message indicates user the token should be a string start with xoxb, I will update the message to be more clear.

File: chatops-update.sh (hunk 2 above)
Comment (yypptest) [reply chatops-update.sh:47]: The PagerDuty API key should be 20-character string, here we provide the checking for the string  length, I will update the message to indicate it should be a 20-character string.

File: chatops-update.sh (hunk 3 above)
Comment (yypptest) [reply chatops-update.sh:60]: Thanks, will update the message.

File: chatops-update.sh (hunk 4 above)
Comment (yypptest) [reply chatops-update.sh:85]: Thanks, will update the message.

**Comment by yypptest** [comment]
@stacyped Thanks for your review, I have updated the scripts with your comments, could you pls help to review again? Thank you.

**Comment by yypptest** [comment]
@stacyped could you pls help to review the updated script? Thank you!

**Review by stacyped — APPROVED** [review]

**Pull request merged**